src/service/transaction.service.ts

```diff
@@ -29,9 +32,9 @@ export const postTransaction = async (req: Request, session: Session): Promise<s
   );
 
   if (!response.httpStatusCode || response.httpStatusCode >= 400) {
-    throw createAndLogErrorRequest(req, `'postTransaction' for company number '${companyNumber}' with name '${companyName}' returned HTTP status code ${response.httpStatusCode}`);
+    throw createAndLogErrorRequest(req, `'>>error:>>>>>>>>postTransaction' for company number '${companyNumber}' with name '${companyName}' returned HTTP status code ${response.httpStatusCode}`);
```

Comment:
The `>>error:>>>>>>>>` text here and below will need to go.

Reply:
Yes, will get rid of these. The [new PR ](https://github.com/companieshouse/overseas-entities-web/pull/1471) doesn't have them.